backend/JavaWebService/src/main/java/com/kainos/ea/models/Role.java

```diff
@@ -9,6 +14,7 @@
     private String location;
     private int bandID;
     private int capabilityID;
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");
```

Comment:
This will be the same for all instances of `Role` so we can make it a constant
```suggestion
    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd-MM-yyyy");
```